extensions/blocks/eventbrite/index.js

```diff
@@ -51,8 +51,8 @@ export const settings = {
 		eventId: {
 			type: 'number',
 		},
-		useModal: {
-			type: 'boolean',
+		style: {
```

Comment:
I think this would break existing blocks using the `useModal` attribute.
We should provide a deprecation/migration path: https://developer.wordpress.org/block-editor/developers/block-api/block-deprecation/

Reply:
Good point, I added a `deprecated` array. Thanks :)